zebra/connected.c

```diff
@@ -423,6 +426,20 @@ void connected_down(struct interface *ifp, struct connected *ifc)
 		break;
 	}
 
+	/* Skip if we've already done this; this can happen if we have a
```

Comment:
I think this hunk belongs at around line 403.  There is no point in doing the work to figure out family and getting `struct prefix p` setup if we are not going to do work

Reply:
sure, I'll push that in